3rdparty/libunwind/stubs.h

```diff
@@ -112,13 +112,13 @@ static __inline void* __libunwind_mmap(
 
 static __inline int __libunwind_munmap(void* addr, size_t length)
 {
-    extern void dlfree(void *ptr);
+    extern void _oe_free(void *ptr);
```

Comment:
This should be oe_free

Reply:
As for the previous comment, this is not `oe_free`, but an internal symbol.